android/envoy/src/main/kotlin/org/greatfire/envoy/NetworkIntentService.kt

```diff
@@ -186,21 +187,55 @@ class NetworkIntentService : IntentService("NetworkIntentService") {
             }
         }
 
-        if (urls.isNullOrEmpty()) {
+        val urlsToSubmit = mutableListOf<String>()
+        if (!urls.isNullOrEmpty()) {
+            urls.forEach { url ->
```

Comment:
I think you can simplify this code with an elvis operator ?:

Reply:
that only catches nulls, right?

Reply:
good point